### ISSUE TITLE
CVE-2021-32629 /  RUSTSEC-2021-0067: Memory access due to code generation flaw in Cranelift module

### ISSUE THREAD

**Issue by jszwedko** [vectordotdev/vector#7558]
Unfortunately this one is deep in lucet so it might not be so easy to update.

```
error[RUSTSEC-2021-0067]: Memory access due to code generation flaw in Cranelift module
    ┌─ /home/jesse/workspace/vector/Cargo.lock:118:1
    │
118 │ cranelift-codegen 0.65.0 git+https://github.com/bytecodealliance/lucet.git?rev=b1863dacc8c92c11e5434fc8815d9b9a26cfe3db#b1863dacc8c92c11e5434fc8815d9b9a26cfe3db
    │ ---------------------------------------------------------------------------------------------------------------------------------------------------------------- security vulnerability detected
    │
    = There is a bug in 0.73.0 of the Cranelift x64 backend that can create a
      scenario that could result in a potential sandbox escape in a WebAssembly
      module. Users of versions 0.73.0 of Cranelift should upgrade to either 0.73.1
      or 0.74 to remediate this vulnerability. Users of Cranelift prior to 0.73.0
      should update to 0.73.1 or 0.74 if they were not using the old default backend.

      More details can be found in the GitHub Security Advisory at:

      <https://github.com/bytecodealliance/wasmtime/security/advisories/GHSA-hpqh-2wqx-7qp5>
    = URL: https://github.com/bytecodealliance/wasmtime/security/advisories/GHSA-hpqh-2wqx-7qp5
    = cranelift-codegen v0.65.0
      ├── cranelift-frontend v0.65.0
      │   ├── cranelift-wasm v0.65.0
      │   │   └── lucetc v0.7.0-dev
      │   │       └── vector v0.14.0
      │   └── lucetc v0.7.0-dev (*)
      ├── cranelift-module v0.65.0
      │   ├── cranelift-object v0.65.0
      │   │   └── lucetc v0.7.0-dev (*)
      │   └── lucetc v0.7.0-dev (*)
      ├── cranelift-native v0.65.0
      │   └── lucetc v0.7.0-dev (*)
      ├── cranelift-object v0.65.0 (*)
      ├── cranelift-wasm v0.65.0 (*)
      └── lucetc v0.7.0-dev (*)
```

**Comment by jszwedko** [comment]
There isn't a fixed version of `lucet` yet so I reached out to them via email as per their security policy.

**Comment by jszwedko** [comment]
Resolved by  #8036
